pkg/controller/servicebindingrequest/controller.go

```diff
@@ -23,8 +23,8 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager, client dynamic.Interface) (reconcile.Reconciler, error) {
+	mgr.GetClient()
```

Comment:
A commend should state the reason this method is being executed for its side-effect.

Reply:
this should be deleted. thanks!